src/routers/router.js

```diff
@@ -5,96 +5,105 @@ import { getExamples, getExample } from '../controllers/examples';
 const router = express.Router();
```

Comment:
It looks like you didn't create an express route for deleting words yet. Inside `editorRouter.js` could you create the delete route like `editorRouter.delete('/words/:id')`?